### PR TITLE
update the pilotsynthea2022q3 config to reflect newer features

### PR DESCRIPTION
**Pull request by amygdala**
Updates the pilotsynthea2022q3 devel config to reflect newer features, based on some guidance from Mariko on what to add/update. 
(There would be more that could be done in future).

### CODE REVIEW

File: underlay/src/main/resources/config/underlay/pilotsynthea2022q3/underlay.json [hunk 1]
```diff
@@ -59,14 +59,25 @@
...
     "pilotsynthea2022q3/race",
     "pilotsynthea2022q3/yearOfBirth",
+    "pilotsynthea2022q3/measurements",
+    "pilotsynthea2022q3/devices",
+    "pilotsynthea2022q3/visits",
+    "pilotsynthea2022q3/cpt4",
+    "pilotsynthea2022q3/icd9CM",
+    "pilotsynthea2022q3/icd9Proc",
+    "pilotsynthea2022q3/icd10CM",
+    "pilotsynthea2022q3/icd10PCS",
```
Comment (amygdala) [inline underlay/src/main/resources/config/underlay/pilotsynthea2022q3/underlay.json:69]: added this line as well.

Comment (marikomedlock) [reply underlay/src/main/resources/config/underlay/pilotsynthea2022q3/underlay.json:69]: These lines reference directories under `config/criteria/pilotsynthea2022q3` that you'll need to add. You can look at the `sd` selector configs as a starting point.

**Comment by amygdala** [comment]
@tjennison-work 
Tim, I believe this is in a pretty good state to merge in.  However, I am far from expert and may have missed something. Would you have time to take a look?

**Review by tjennison-work — APPROVED** [review]
Looks good to me. Thanks for updating this.

**Comment by amygdala** [comment]
thanks, @tjennison-work !

**Pull request merged**